qemu/tests/virtio_console.py

```diff
@@ -280,7 +280,7 @@ def test_rw_host_offline_big_data():
         guest_worker.cmd("virt.clean_port('%s'),1024" % port.name, 10)
         match, tmp = guest_worker._cmd("virt.send('%s', (1024**3)*3, True, "
                                        "is_static=True)" % port.name, 30)
-        if match is None:
+        if match is not None:
```

Comment:
@sitoliu 
Tested this patch. Case test fail.
# python ConfigTest.py --testcase=virtio_console.spread_2.specifiable.virtserialport.virtio_console_smoke.rw_host_offline_big_data --guestname=RHEL.7.4 --platform=x86_64 --driveformat=virtio_blk --clone=no
......
TestFail: test_rw_host_offline_big_data failed: Write on guest while host disconnected didn't time out